prov/rxm/src/rxm_cq.c

```diff
@@ -236,7 +236,6 @@ static int rxm_lmt_tx_finish(struct rxm_tx_entry *tx_entry)
 	if (!OFI_CHECK_MR_LOCAL(tx_entry->ep->rxm_info))
 		rxm_ep_msg_mr_closev(tx_entry->mr, tx_entry->count);
 
-	tx_entry->comp_flags |= FI_SEND;
```

Comment:
Is this flag being lost?

Reply:
This flag is actually redundant since it would be added when posting the send. I'll add a note in the commit message.